v22.1/api-support-policy.md

```diff
@@ -0,0 +1,108 @@
+
+### Stability guarantees
+
+An API is *stable* if it does not receive backward-incompatible changes. API stability implies programmability.
+
+The following stability guarantees apply to **programmable** APIs. These APIs remain stable within major versions:
+
+- Backward-incompatible changes may be introduced in new major versions.
+- Backward-compatible changes may be introduced in new minor versions.
+
+Stability guarantees do **not** apply to non-programmable and reserved APIs:
+
+- Backward-incompatible changes may be introduced in new minor and major versions.
+
+#### API changes
+
+The following types of API changes qualify as *backward-incompatible* (or "breaking changes"):
+
+- Removal of an endpoint, SQL statement, [built-in function](functions-and-operators.html#built-in-functions), [cluster setting](cluster-settings.html), or session variable.
+- Addition or removal of a mandatory or optional command-line flag.
+- Change in SQL statement syntax, HTTP request format, HTTP response format, or structured log format.
+- Change in data type, default value, or behavior of a [built-in function](functions-and-operators.html#built-in-functions), [cluster setting](cluster-settings.html), or session variable.
+- Change to [authorization](security-reference/authorization.html) requirements.
+
+The following types of API changes qualify as *backward-compatible*. This list is not exhaustive.
+
+- Removal or change of any functionality documented or named as "experimental", "beta", or otherwise not fully supported.
+- Deprecation of any functionality without removal.
+- Addition or removal of a metric.
+- Change in SQL response format or unstructured log format.
+
+{{site.data.alerts.callout_success}}
+Backward-incompatible changes to CockroachDB, including those affecting programmable APIs, are identified in the [release notes](../releases/index.html#production-releases) for major CockroachDB versions. Users are asked to [consider backward-incompatible changes before upgrading](upgrade-cockroach-version.html#review-breaking-changes) to a new CockroachDB version.
+{{site.data.alerts.end}}
+
+#### Versioning
+
+A programmable API may be assigned a new version number in two situations:
+
+- When changes are introduced to the API.
+- When a new CockroachDB version is released.
+
+The assignment of a **major** version number to a [programmable API](#support-policies) indicates that the API is stable within that major version.
+
+## APIs
+
+{{site.data.alerts.callout_info}}
+A *mixed* API includes both programmable and non-programmable features. [Stability guarantees](#stability-guarantees) apply to the programmable features.
+{{site.data.alerts.end}}
+
+### CockroachDB
+
+These interfaces are hosted by the CockroachDB application binary. They are available in {{ site.data.products.serverless }}, {{ site.data.products.dedicated }}, and {{ site.data.products.core }}.
```

Comment:
I don't think all of these are available in Serverless, for example, Cluster API and logging. Maybe not in Dedicated, too. Can we double-check?